istioctl/cmd/wait.go

```diff
@@ -136,8 +136,8 @@ func waitCmd() *cobra.Command {
 		"The duration to wait before failing")
 	cmd.PersistentFlags().Float32Var(&threshold, "threshold", 1,
 		"The ratio of distribution required for success")
-	cmd.PersistentFlags().StringVar(&resourceVersion, "resource-version", "",
-		"Wait for a specific version of config to become current, rather than using whatever is latest in "+
+	cmd.PersistentFlags().StringVar(&generation, "generation", "",
```

Comment:
The problem is that passing an actual resource version will result in unexpected behavior.  Since this is an experimental command, I don't think we have any commitment to backwards compatibility.

Reply:
This needs a release note at the very least that its been broken. Possibly a warning telling you to migrate to generation when you use resourceVersion would help?